.build/ja/index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="ja" prefix="og: http://ogp.me/ns#">
+  <head>
+    <meta charset="utf-8">
+    <title>CashShuffle  -  Bitcoin Cashのためのプライバシー</title>
```

Comment:
Yeah a few places with these double spaces. Probably should normalize them.

Reply:
Wait. Why is the build in the repo?

Reply:
Thanks for explaining offline. I rebuilt for production `grunt live` and `rebuild.sh -p` and pushed the changes.